### ISSUE TITLE
Bug: c2wwise forces plugin version to correspond to Wwise version

### ISSUE THREAD

**Issue by eu-ch** [dromer/hvcc#142]
bundle_template.json sets `targetWwiseVersion` to be the same as plugin version, but it should be set based on parsing `AkWwiseSDKVersion.h` or `$WWISEROOT/install-entry.json`

```json
    "id": "Heavy.{{name}}.${year}_${major}_${minor}_${build}",
    "name": "{{name}}",
    "tag": "{{name}}",
    "description": "",
    "vendor": "Heavy",
    "image": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=",
    "type": "plugin",
    "productDependentData": {
        "targetWwiseVersion": {
            "year": ${year},
            "major": ${major}
        }
    },

```

**Comment by eu-ch** [comment]
Copying my answer from discord. I can get version from certain files in the Wwise SDK folder (I've mentioned two possible options in the ticket). Bundle template us used for making installable bundles and it requires an image, so here you go 🙂 I can possibly include a hvcc logo there instead

bundle_template itself is turned into a bundle.json when making a package

**Comment by dromer** [comment]
Ok, so this bundle template is used by the wwise SDK scripts, not by c2wwise?

**Comment by eu-ch** [comment]
> Ok, so this bundle template is used by the wwise SDK scripts, not by c2wwise?

Correct. I just want making plugin bundles as frictionless as possible. It's a really useful feature of new build system and I liked using it in past projects.

I can just check do of files under `$WWISEROOT` of what's installed and fall back to using current setup. See `install-entry.json` under Wwise installation, it contains all metadata about installed SDK packages.
